pkg/backup/backup/backup_cleaner.go

```diff
@@ -218,6 +285,205 @@ func (bc *backupCleaner) makeCleanJob(backup *v1alpha1.Backup) (*batchv1.Job, st
 	return job, "", nil
 }
 
+func (bc *backupCleaner) makeStopLogBackupJob(backup *v1alpha1.Backup) (*batchv1.Job, string, error) {
```

Comment:
is it possible to re-use `makeBRBackupJob`?

Reply:
The key problem is they have different caller. backupCleaner can't call method of backupManager, and it is weird to use backupCleaner to build other tasks.

Reply:
How about reuse `makeCleanJob`? Their main difference is parameter.

Reply:
I prefer leave it be, they are actually using different pods.
I tried to combine them into one, but that only makes the function fat and mess.